https://gitlab.gnome.org/GNOME/gnome-tweaks/-/merge_requests/86.patch
https://bugs.gentoo.org/831937

From 56f39dc7d04a1be91de1149683874f9c83b794fb Mon Sep 17 00:00:00 2001
From: Mae Dartmann <mae@meanderingmind.me>
Date: Sat, 22 Jan 2022 18:09:21 +0100
Subject: [PATCH] meson: fix invalid positional argument

This was deprecated in meson 0.60 and removed in 0.61. See
https://github.com/mesonbuild/meson/pull/9445 for details.

Signed-off-by: Mae Dartmann <mae@meanderingmind.me>
--- a/data/meson.build
+++ b/data/meson.build
@@ -1,5 +1,5 @@
 appdata_file = 'org.gnome.tweaks.appdata.xml'
-i18n.merge_file(appdata_file,
+i18n.merge_file(
   input: appdata_file + '.in',
   output: appdata_file,
   po_dir: '../po',
@@ -8,7 +8,7 @@ i18n.merge_file(appdata_file,
 )
 
 desktop_file = 'org.gnome.tweaks.desktop'
-i18n.merge_file(desktop_file,
+i18n.merge_file(
   input: desktop_file + '.in',
   output: desktop_file,
   po_dir: '../po',
GitLab
